### DISCUSSION TITLE
warning:Body property 'CollectingDate' not defined in body schema (Though CollectingDate is defined in schema) what could be the possible reason for this warning?

### DISCUSSION THREAD

**Discussion by Ragzz1995** [OAI/OpenAPI-Specification#2778] [Q&A]
I have defined CollectingDate property in my swagger.yaml file, but whenever I make put request I get **Body property 'CollectingDate' not defined in body schema** What could be the possible reason?

**Comment by webron** [comment]
It's really impossible to tell without seeing your definition file. It could also be a tooling issue, but again, missing details...

**Comment by handrews** [comment]
No reply (for over 2 years), seems like a tooling question anyway, closing.
